php/Log/NullLogger.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace PHPCD\Log;
+
+use Psr\Log\AbstractLogger;
+
+class NullLogger extends AbstractLogger
+{
+    /**
+     * Does nothing
+     *
+     * @param mixed  $level
+     * @param string $message
+     * @param array  $context
+     *
+     * @return null
+     */
+    public function log($level, $message, array $context = [])
+    {
+        return null;
```

Comment:
No, it is especially need - even I would like to propose it as default logger when application works properly and then user does not want to log anything.